code/game/objects/items/weapons/grenades/clusterbuster.dm

```diff
@@ -252,6 +252,10 @@
 	desc = "For when you need to knock out EVERYONE."
 	payload = /obj/item/weapon/grenade/gas/knockout
 
+/obj/item/weapon/grenade/clusterbuster/glass
+	name = "Glass Grenade"
+	desc = "Useful for impaling EVERYONE." // Coders have worse grammer than spriting ability. Thats me!
+	payload = /obj/item/weapon/shard
```

Comment:
These don't actually embed, since clusterbusters use `walk_away` instead of `throw_at` - the most this will do is be crowd control against shoeless people